generator/templates/wire/WireServer.cpp

```diff
@@ -262,20 +351,19 @@ namespace wire {
                 //* Checks there is enough data left, updates the buffer / size and returns
                 //* the command (or nullptr for an error).
                 template<typename T>
-                static const T* GetCommand(const uint8_t** commands, size_t* size) {
+                static const T* GetCommand(const char** commands, size_t* size) {
                     if (*size < sizeof(T)) {
                         return nullptr;
                     }
 
-                    const T* cmd = reinterpret_cast<const T*>(*commands);
-
-                    size_t cmdSize = cmd->GetRequiredSize();
-                    if (*size < cmdSize) {
+                    if (*size < sizeof(T)) {
```

Comment:
This is the same as the if() above this.

Reply:
Done